### PR TITLE
feat: Add more flexible OpenRouteService configuration to support self-hosting

### PR DESCRIPTION
**Pull request by joshlarson**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206910009181796

### CODE REVIEW

**Comment by joshlarson** [comment]
Converted back to draft because we need the OPEN_ROUTE_SERVICE_DIRECTIONS_PATH to be defined in all environments before we merge this!

**Comment by firestack** [comment]
> Converted back to draft because we need the OPEN_ROUTE_SERVICE_DIRECTIONS_PATH to be defined in all environments before we merge this!

Would it be worth falling back to the old directions path for now in the config file and then remove that code in a later PR?

**Comment by joshlarson** [comment]
>> Converted back to draft because we need the `OPEN_ROUTE_SERVICE_DIRECTIONS_PATH` to be defined in all environments before we merge this!
>
> Would it be worth falling back to the old directions path for now in the config file and then remove that code in a later PR?

Good call! Now this doesn't need to be blocked on https://github.com/mbta/devops/pull/1642

**Review by firestack — APPROVED** [review]

**Pull request merged**
